todo/services/task_service.py

```diff
@@ -173,6 +186,7 @@ def create_task(cls, dto: CreateTaskDTO) -> CreateTaskResponse:
                 )
 
         task = TaskModel(
+            id=None,
```

Comment:
- is this change required? I do not see any changes in this task model in this pr.